resources/views/base.blade.php

```diff
@@ -168,7 +168,7 @@ class="postal-code">55455</span></div>
   window.pusherKey = '{{ env('PUSHER_APP_KEY') }}'
 </script>
 <script>
-  window.mapbox = "{{ config('services.mapbox') }}";
+  window.mapbox = "{{ env('MIX_MAPBOX_ACCESS_TOKEN') }}";
```

Comment:
what's the reason for this change? (Just curious, nbd either way)

Reply:
I'll revert – it's not needed in this PR.

This was changed in the course of troubleshooting why tiles weren't rendering for mapbox sometimes. Turns out we have two env variables for mapbox and I was missing one. `config('services.mapbox')` added another layer, so it got simplified while debugging/

Ultimately, I'd like to just have one env variable, unless a separate one is needed for server side, but that's for later.